### ISSUE TITLE
Can't install the hfc by "npm install -g hfc"

### ISSUE THREAD

**Issue by huhaiqing** [hyperledger-archives/fabric#2251]
Hi,

When use the command 'npm install -g hfc', it returns below errors.

npm ERR! not a package /tmp/npm-1395-e1j7AUS3/github.com/mcavage/node-asn1
npm ERR! Error: ENOENT, open '/tmp/npm-1395-e1j7AUS3/github.com/mcavage/node-asn1-unpack/package.json'
npm ERR! If you need help, you may report this _entire_ log,
npm ERR! including the npm and node versions, at:
npm ERR!     http://github.com/npm/npm/issues

npm ERR! System Linux 4.4.0-31-generic
npm ERR! command "/usr/bin/nodejs" "/usr/bin/npm" "install" "-g" "hfc"
npm ERR! cwd /
npm ERR! node -v v0.10.29
npm ERR! npm -v 1.4.21
npm ERR! path /tmp/npm-1395-e1j7AUS3/github.com/mcavage/node-asn1-unpack/package.json
npm ERR! code ENOENT
npm ERR! errno 34
npm ERR!
npm ERR! Additional logging details can be found in:
npm ERR!     /npm-debug.log
npm ERR! not ok code 0

Could you help how to install the hfc?

Thanks.

Regards
Harry

**Comment by timjacobi** [comment]
Your npm and node versions are quite old. Could you try to update?

**Comment by smithbk** [comment]
Also try "sudo npm install -g hfc" as you'll need this for npm to install with "-g"

**Comment by timjacobi** [comment]
@smithbk `sudo npm install` [should be avoided](https://docs.npmjs.com/getting-started/installing-npm-packages-globally)

**Comment by smithbk** [comment]
@timjacobi good point.  You could use https://github.com/sindresorhus/guides/blob/master/npm-global-without-sudo.md and still be able to run 'hfc' from command line.
